app/react/settings/people/PeopleToolbarContainer.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Link from "../../react-transition/Link";
+
+const PeopleToolbarContainer = () => (
```

Comment:
Should we make this a reusable component now, or refactor when we need it next?

Reply:
this will probably go out with the new designs, I'd refactor into generic after we start getting them